tools/check-for-update.js

```diff
@@ -1,41 +1,55 @@
 const fetch = require('node-fetch');
 const fs = require('fs');
 const semver = require('semver');
-const util = require('util');
 
-const childProcess = require('child_process');
-const exec = util.promisify(childProcess.exec);
+const exec = require('child_process').exec;
 
 let pkg = JSON.parse(fs.readFileSync('package.json', {encoding: 'utf-8'}));
 
-fetch('https://api.github.com/repos/mozilla-iot/gateway/releases').then(res => {
+fetch('https://api.github.com/repos/hobinjk/gateway/releases').then(res => {
```

Comment:
Perhaps the root of the repo should go into a variable? That way there is only one place to change.

Presumably having this point to hobinjk is just temporary?

Reply:
I've factored it out into a variable to make changing it easier. In the current revision it should be back to mozilla-iot